lib/pbench/server/__init__.py

```diff
@@ -191,15 +194,39 @@ def max_retention_period(self) -> int:
         allows a dataset to be retained.
 
         Returns:
-            integer representing retention period in days
+            An integer number of days representing the maximum retention period.
         """
-        try:
-            retention_days = int(
-                self._get_conf("pbench-server", "maximum-dataset-retention-days")
-            )
-        except (NoOptionError, NoSectionError):
-            retention_days = self.MAXIMUM_RETENTION_DAYS
-        return retention_days
+        return self.conf.getint(
+            "pbench-server",
+            "maximum-dataset-retention-days",
+            fallback=self.MAXIMUM_RETENTION_DAYS,
+        )
+
+    @property
+    def default_retention_period(self) -> int:
+        """Produce an integer representing the default number of days the server
+        allows a dataset to be retained.
+
+        Returns:
+            An integer number of days representing the default retention period.
+        """
+        return self.conf.getint(
+            "pbench-server",
+            "default-dataset-retention-days",
+            fallback=self.DEFAULT_RETENTION_DAYS,
+        )
+
+    @property
+    def rest_max_content_length(self) -> int:
+        """Produce an integer representing the maximum content length accepted
+        by the REST APIs.
+
+        Returns:
+            An integer number of bytes.
+        """
+        return filesize_bytes(
+            self.conf.get("pbench-server", "rest_max_content_length", fallback="1 gb")
+        )
```

Comment:
Are we sure we want the Server to export/enforce this limitation?  (E.g., don't we want to implement that in the proxy?)  And, `1 gb` seems low to me...no?

Reply:
Good question.  I don't think we want the server to enforce this.

I moved this to a property like the others since it seemed silly to leave this one as is when I modified the other two.

I'd rather address that is another PR (which I thought somebody was already doing).

Reply:
There's a Jira for it; I don't think anyone's actually doing it now. (Which is good, since dealing with a merge conflict on removing a limit because you're changing the implementation of that limit would just be silly.)